docs/USAGE-GUIDE.md

```diff
@@ -343,6 +343,33 @@ An application can override s2n-tls’s internal memory management by calling `s
 
 If you are trying to use FIPS mode, you must enable FIPS in your libcrypto library (probably by calling `FIPS_mode_set(1)`) before calling `s2n_init()`.
 
+## Connection
+
+Users will need to create a `s2n_connection` struct to store all of the state necessary to negotiate a TLS connection. Call `s2n_connection_new()` to create a new server or client connection. Call `s2n_connection_free()` to free the memory allocated for this struct when no longer needed.
```

Comment:
Oh, sorry I didn't catch this the first time. "negotiate" might be misleading, since we need to keep it around after the negotiation / handshake.
```suggestion

Users will need to create a `s2n_connection` struct to store all of the state necessary for a TLS connection. Call `s2n_connection_new()` to create a new server or client connection. Call `s2n_connection_free()` to free the memory allocated for this struct when no longer needed.
```